docker-compose.yml

```diff
@@ -3,37 +3,54 @@ version: "3.7"
 services:
   postgres:
     image: postgres:15
-    expose:
-      - 5432
-    environment:
-      - "POSTGRES_USER=ayon"
-      - "POSTGRES_PASSWORD=ayon"
-      - "POSTGRES_DB=ayon"
-    volumes:
-      - "/etc/localtime:/etc/localtime:ro"
-      - "db:/var/lib/postgresql/data"
     restart: unless-stopped
     healthcheck:
       test: ["CMD-SHELL", "pg_isready -U ayon"]
       interval: 5s
       timeout: 5s
       retries: 5
 
+    expose: [5432]
+
+    volumes:
+      - "/etc/localtime:/etc/localtime:ro"
+      - "db:/var/lib/postgresql/data"
+
+    environment:
+      - "POSTGRES_USER=ayon"
+      - "POSTGRES_PASSWORD=ayon"
+      - "POSTGRES_DB=ayon"
+
   redis:
     image: redis:alpine
-    expose:
-      - 6379
     restart: unless-stopped
-    # TODO: This check floods logs (too many writes)
-    # healthcheck:
-    #   test: ["CMD", "redis-cli", "--raw", "incr", "ping"]
-    #   interval: 5s
-    #   timeout: 5s
-    #   retries: 5
+    healthcheck:
+      test: ["CMD-SHELL", "redis-cli ping | grep PONG"]
+      interval: 5s
+      timeout: 5s
+      retries: 5
+
+    expose: [6379]
+
 
   server:
     image: ynput/ayon:dev
     restart: unless-stopped
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://localhost:5000/api/info"]
+      interval: 10s 
+      timeout: 2s
+      retries: 3
+
+    depends_on:
+      postgres:
+        condition: service_healthy
+      redis:
+        condition: service_started
+
+    expose: [5000]
+    ports: ["5000:5000"]
+
     volumes:
       - "./addons:/addons"
```

Comment:
I think that in systems with `selinux` enabled, you need to add ` :Z` at the end, ie:
`"./addons:/addons:Z"`

But i seem to be the only one, since no one else complained :joy: